browser/ui/views/tabs/tab_drag_controller.cc

```diff
@@ -83,24 +83,19 @@ gfx::Point TabDragController::GetAttachedDragPoint(
 
 void TabDragController::MoveAttached(const gfx::Point& point_in_screen,
                                      bool just_attached) {
-  TabDragControllerChromium::MoveAttached(point_in_screen, just_attached);
   if (!is_showing_vertical_tabs_) {
+    TabDragControllerChromium::MoveAttached(point_in_screen, just_attached);
```

Comment:
I think this call is the same in both paths? Maybe it can be moved back out?

Reply:
Oh yeah, you're right.